Application/Api/Billing/Bookkeeping/ApiBasketRepaymentAddPerson.php

```diff
@@ -0,0 +1,903 @@
+<?php
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5646354)